tests/conftest.py

```diff
@@ -307,85 +290,281 @@ def simple_inventory_node() -> None:
+    algorithm = cript.Algorithm("mc_barostat", "barostat")
+    return algorithm
 
 
-# ---------- Supporting Nodes ----------
 @pytest.fixture(scope="function")
-def simple_file_node() -> cript.File:
-    """
-    simple file node with only required arguments
-    """
-    my_file = cript.File(source="https://criptapp.org", type="calibration", extension=".csv", data_dictionary="my file's data dictionary")
+def simple_algorithm_dict() -> dict:
+    ret_dict = {"node": ["Algorithm"], "key": "mc_barostat", "type": "barostat"}
+    return ret_dict
 
-    return my_file
+
+@pytest.fixture(scope="function")
+def simple_reference_node() -> cript.Reference:
+    title = "Multi-architecture Monte-Carlo (MC) simulation of soft coarse-grained polymeric materials: "
+    title += "SOft coarse grained Monte-Carlo Acceleration (SOMA)"
+
+    reference = cript.Reference(
+        "journal_article",
+        title=title,
+        authors=["Ludwig Schneider", "Marcus Müller"],
+        journal="Computer Physics Communications",
+        publisher="Elsevier",
+        year=2019,
+        pages=[463, 476],
+        doi="10.1016/j.cpc.2018.08.011",
+        issn="0010-4655",
+        website="https://www.sciencedirect.com/science/article/pii/S0010465518303072",
+    )
+    return reference
+
+
+@pytest.fixture(scope="function")
+def simple_reference_dict() -> dict:
+    ret_dict = {
+        "node": ["Reference"],
+        "type": "journal_article",
+        "title": "Multi-architecture Monte-Carlo (MC) simulation of soft coarse-grained polymeric materials: SOft coarse grained Monte-Carlo Acceleration (SOMA)",
+        "authors": ["Ludwig Schneider", "Marcus Müller"],
+        "journal": "Computer Physics Communications",
+        "publisher": "Elsevier",
+        "year": 2019,
+        "pages": [463, 476],
+        "doi": "10.1016/j.cpc.2018.08.011",
+        "issn": "0010-4655",
+        "website": "https://www.sciencedirect.com/science/article/pii/S0010465518303072",
+    }
+    return ret_dict
 
 
 @pytest.fixture(scope="function")
 def simple_citation_node(simple_reference_node) -> cript.Citation:
-    """
-    minimal citation node
-    """
-    my_citation = cript.Citation(type="derived_from", reference=simple_reference_node)
+    citation = cript.Citation("reference", simple_reference_node)
+    return citation
+
 
-    return my_citation
+@pytest.fixture(scope="function")
+def simple_citation_dict(simple_reference_dict) -> dict:
+    ret_dict = {"node": ["Citation"], "reference": simple_reference_dict, "type": "reference"}
+    return ret_dict
 
 
 @pytest.fixture(scope="function")
 def simple_quantity_node() -> cript.Quantity:
-    """
-    minimal quantity node
-    """
-    my_quantity = cript.Quantity(key="mass", value=1.23, unit="gram")
+    quantity = cript.Quantity("mass", 11.2, "kg", 0.2, "std")
+    return quantity
 
-    return my_quantity
+
+@pytest.fixture(scope="function")
+def simple_quantity_dict() -> dict:
```

Comment:
oh I see so you're making the node fixture and then you're making the dict that would be if it were to be serialized.

thats not a bad idea, so updating the fixture and the dict would be easier and more maintainable, but I would want to start separating out the files at some point to keep primary nodes in one file, dicts in another, and so on because otherwise we are just throwing everything into one file and the file will become unruly